cumulus/parachains/runtimes/assets/test-utils/src/lib.rs

```diff
@@ -19,4 +19,63 @@
 pub mod test_cases;
 pub mod test_cases_over_bridge;
 pub mod xcm_helpers;
+
+use frame_support::traits::ProcessMessageError;
 pub use parachains_runtimes_test_utils::*;
+use std::fmt::Debug;
+
+use xcm::latest::prelude::*;
+use xcm_builder::{CreateMatcher, MatchXcm};
+
+/// Given a message, a sender, and a destination, it returns the delivery fees
+fn get_fungible_delivery_fees<S: SendXcm>(destination: MultiLocation, message: Xcm<()>) -> u128 {
```

Comment:
I think this is not used anywhere, right?

Reply:
used [in test_cases.rs](https://github.com/paritytech/polkadot-sdk/blob/f838cd51b20ef59e104d2e42e817acdc3323169f/cumulus/parachains/runtimes/assets/test-utils/src/test_cases.rs#L1538)